Replace java based self-signed certificate generation with OpenSSL (#147)
neo4j/docs-ops-manager: modules/ROOT/pages/first-look/docker-first-look.adoc

==  Create persistence DBMS and NOM server

=== Generate self-signed certificates

. Create a directory *`~/.nom/ssc`* on your local machine before running Docker compose.
The compose files below specify that this directory will be mounted into the docker container to make the generated certificates available to the NOM server and the agent.
+
[NOTE]
====
For *`sudo`* docker compose command, make sure the correct path to *`.nom/ssc`* is set in the bind section below.
====

> . Change to *`~/.nom/ssc`* and generate self-signed certificates as described *xref:installation/self-signed-certificate.adoc[here]*, using `server` as common name (CN) and DNS name and `127.0.0.1` as IP address.

Reviewer: ```suggestion
. Change to *`~/.nom/ssc`* and generate self-signed certificates as described *xref:installation/self-signed-certificate.adoc[here]*, using `server` as common name (CN) and DNS name, and `127.0.0.1` as IP address.
```

> This will result in `server.cer` and `server.pfx` being created.

Reviewer: ```suggestion
This results in the creation of `server.cer` and `server.pfx`.
```
